client/src/components/Predictions.jsx

```diff
@@ -21,9 +22,15 @@ const Prediction = (props) => {
 const Predictions = (props) => {
   return props.predictions.map((prediction, i) => {
     return (
+<<<<<<< HEAD
       //<Link to='/stock-detail-page'>
         <Prediction key={i} data={prediction} predictionClick={props.predictionClick}/>
       //</Link>
+=======
+      <Link to="/stock-detail-page">
+      <Prediction key={i} data={prediction} predictionClick={props.predictionClick}/>
+      </Link>
+>>>>>>> 3d3eab71d706dce1c74a86301d7fc72d3113d220
```

Comment:
merge leftovers...

Reply:
might need to check the most recent version of development to figure out which section to keep

Reply:
Yeah. I'll get to the bottom of this.